### ISSUE TITLE
eBUS adapter 3 HW fault?

### ISSUE THREAD

**Issue by pintergabee** [eBUS/adapter#14]
I have one of your ebus adapter 3, which doesn't work as expected unfortunately. In fact the communication with the adapter either fails or it is not deterministic. I would like to request some help how can I verify, what is the root cause, because in this form the adapter is not functional.

The symptoms are following:

1. Anyhow I connect the adapter, the blue led light up brightly and the orange too but not so bright. Both of them are static.

2. Firmware flashing 
Setup: adapter configured for USB, connected to a raspberry pi 4 via /dev/ttyUSB0, jumper is up for bootloader mode
- Setting of dhcp option seems to work - there is some sort of communication with the adapter
- Flashing of FW fails at different points.
- Note the unable to lock messages. I can overcome on this by moving the USB cable between the USB ports of Raspberry. (Works only for one command, after that I have to change to USB port again)

![unnamed](https://user-images.githubusercontent.com/4817559/196913520-3f611602-7c25-496e-9a39-368d2f5fad8c.png)

3. Connect adapter to raspberry through UART
Setup: J1, J4 configured to RPI, bootloader jumper on, adapter put on rpi UART pins
- same constant state of adapter leds: bright blue, not so bright orange
- rpi looses all connection upon connecting the adapter! No IP, ssh session closes
- upon re-powering rpi it does not come up, does not get IP --> UART mode doesn't work at all
- I did all the settings in raspi-config required to have UART mode working

4. Connect adapter to Wemos D1 mini:
Setup: J1, J4, J12 set as required, D1 mini flashed with latest firmware, ebusd-esp GUi is available, adapter connected to D1 mini, which is powered. ebusd runs on a different host as a service and logs to file, adapter is not connected to bus.

![unnamed](https://user-images.githubusercontent.com/4817559/196913639-b2577b23-12f0-46db-bbe6-3057d1c9ce5b.png)
![unnamed](https://user-images.githubusercontent.com/4817559/196913655-82619945-188b-44ec-8e11-c5a0c07edf0a.png)

So basically I tried 3 out of the 4 possible main HW setups and non of them are working. I don't think I miss something in each case, but certainly it may happen.
Could you please advise how can I move forward? Maybe to verify somehow the hardware is fine or to move out from a stucked state (again, leds are always showing the same static pattern...)

Thanks in advance!

**Comment by john30** [comment]
as discussed via email. furthermore: `unable to lock` clearly indicates, that another process is using the device already. this way it can't be flashed of cource.
